muffnn/autoencoder/tests/test_autoencoder.py

```diff
@@ -505,48 +568,49 @@ def test_sparse_inputs():
     for sparse_type in [None, 'csr', 'bsr', 'coo', 'csc', 'dok', 'lil']:
         scores.append(_check_ae(
             5.0,
+            hidden_units=(2,),
             sparse_type=sparse_type,
             cat_inds=range(4),
-            n_epochs=1000))
+            n_epochs=1000,
+            learning_rate=5e-3))
```

Comment:
I upped the learning rate here to make sure the autoencoder converged. Not needed for the tests per se, but def useful to make sure it could. I will leave a comment.